api/groups/interface.go

```diff
@@ -9,6 +9,7 @@ import (
 // EventsFacadeHandler defines the behavior of a facade handler needed for events group
 type EventsFacadeHandler interface {
 	HandlePushEvents(events data.ArgsSaveBlockData) error
+	HandlePushEventsOld(events data.SaveBlockData) error
```

Comment:
or we could have named this `HandlePushEventsV1` and the newer one: `HandlePushEventsV2`